Lib/test/test_zipfile.py

```diff
@@ -1442,6 +1442,16 @@ def test_read0(self):
 
                 self.assertEqual(f.read(), b"O, for a Muse of Fire!")
 
+    def test_seek_before_current_position(self):
```

Comment:
Can the existing test `test_seek_tell` be modified to cover this case?

Reply:
Ah yes, done, thank you!